cylc/rose/entry_points.py

```diff
@@ -103,13 +103,20 @@ def get_rose_vars(srcdir=None, opts=None):
 
     # Load the raw config tree
     config_tree = rose_config_tree_loader(srcdir, opts)
-
-    # Warn if root-dir set in config:
-    if 'root-dir' in config_tree.node:
-        LOG.warning(
-            'You have set "root-dir", which at Cylc 8 does nothing. '
-            'See Cylc Install documentation.'
-        )
+    # Warn if root-dir set in config
+    old_rose_suite_run_config = (
+        config_tree.node.get_value(["rose-suite-run"]))
```

Comment:
Just to confuse you the `root-dir` setting goes in a `rose-suite-run` section in the site-user config but *not* in the suite config.

* http://metomi.github.io/rose/doc/html/api/configuration/site-and-user.html#rose:conf:rose.conf[rose-suite-run]root-dir
* http://metomi.github.io/rose/doc/html/api/configuration/suite.html#rose:conf:rose-suite.conf|root-dir

Reply:
Ah doh. I'll change this. Want me to change this to check the site-user config or not check?

Reply:
I think we can ignore the site-user config.